---
title: 'Oscillating Organoids: Second Research Paper Published!'
tags: [Reflections, Mind & Brain, Science Communication]
status: publish
type: post
published: true
header:
  overlay_image: /assets/images/blog/2019-09-03-mea.png
  overlay_filter: rgba(0,0,0,0.7)
  teaser: /assets/images/blog/2019-09-03-mea.png

excerpt: "In hindsight, there was some bad luck, and also mistakes on our part. But by and large, and only because it is now officially over, I can say that it was a great learning experience. I shit you not, I had a dream one night that the organoid grew to be like a boulder-sized ball and it was literally rolling around chasing after me."
---

Every now and then my mom tells me that I'm my parents' biggest accomplishment, but I'm not sure if that's the whole truth. If I had to guess, I'd say they were ecstatic when I was between the ages of 0-12. Then, slowly, I transformed into the most insufferable almost-human being for 6 to 8 long years between high school and early undergrad, during which time they must have thought to themselves, "this was a huge mistake." Then, all of a sudden, this shit kid graduated from college and is starting a PhD, and doesn't seem to be a generally horrible person (anymore), so maybe it wasn't so bad after all.

I don't know if that's how my parents would talk about raising me, but that's exactly how I feel about this demon-child of a paper, which **finally** came out in [Cell Stem Cell][trujillo2019] last week. After being numbed by a whole year of onslaught from reviewers and rejections, apparently I blinked too fast and it got published last week while I was away stuffing myself with pasteis de nata in Lisbon.

In this blog post, I recount the 5-year saga of this project. I think it's a neat story about some super interesting science, but also personally tragic, and at the same time a pretty insightful look into science and interdisciplinary research. If you don't want to read the behind-the-scenes story, skip to the **2016-2017** and **2018-2019** sections for an explanation of the key analyses. Alternatively, read the very compelling yet concise and accurate [New York Times][nyt] piece that explains the main results without technical detail. But I promise you, the story is worth it, and it's uncensored.


After my first paper, I didn't think that it was possible to have a harder time publishing. Boy was I wrong. In hindsight, there was some bad luck, and also mistakes on our part. But by and large, and **only** because it is now officially over, I can say that it was a great learning experience. I shit you not, I had a dream one night that the organoid grew to be like a boulder-sized ball and it was literally rolling around chasing after me. Just to give you a sense of our collective relief, these are the texts I got from Cleber and Brad minutes after we heard it was finally accepted back in early July:

![][itsovercleber]
![][itsoverbrad]


**TL;DR**: we (by which I mean Pri and Cleber) cultured stem cell-derived brain organoids, which developed spontaneous 2-3Hz oscillations that matured over time. These oscillations are prominent in most mammalian organisms but previously unseen in in-vitro cultures (not counting slices preps). If you **squint really hard**, through the lens of a simple regularized regression model, the developmental trajectory of those electrical features looks kinda similar to that of prematurely born human infants around the same time period (25-38 weeks). And no, we're not building battle robots with them, and I really don't think they are more conscious than a piece of booger, which is exactly what they look like, but we'll delve into that in more detail as well at the end.

---
### 2014-2016: The 2D Cultures and Salad Fingers

The story began in September 2014, almost exactly 5 years ago. During my very first PhD meeting with Brad, he told me there is a group on campus that takes human skin cells and reprograms them into stem cells (induced pluoripotent stem cells, iPSCs), and then turns those into neurons. They reached out to Brad, who had just started our lab, because they wanted to compare the electrical activity of the iPSC-derived neurons from children with various developmental disorders with their own EEGs recordings, as a better model of, for example, autism. He asked if I wanted to work with them, and I was like, human brain tissue grown from their own cells, with their own DNA, and compared to their own brain waves? You're shitting me right? This is literal magic. I had no idea that it was even possible to turn differentiated cells back into stem cells, that seemed like a Nobel prize award for sure (in fact, [it was, in 2012][nobel]). Well, I couldn't say no to that, and so began our collaboration with Alysson Muotri's lab at the Sanford Consortium.

[nobel]: https://www.eurostemcell.org/nobel-prize-goes-stem-cell-pioneers-john-b-gurdon-and-shinya-yamanaka

I started working with a PhD student in his lab at the time, Jerome, who later became a really close friend. During those first two years, we weren't using organoids yet, but the technology was emerging ([Lancaster 2013][lancaster2013]). Jerome was growing 2D cultures from human iPSC-derived neurons, and later, neurospheres. People have been working with 2D cultures for ages, mostly using neurons extracted from mouse brains right around birth by scooping out a bit of brain tissue, which has already formed networks, and dissolving them and plating them on a petri dish to reestablish naive networks. The innovation here is that, obviously, we were using human cells that start out as stem cells. Essentially, you take the iPSCs, plate them, feed with various growth factors, and they become neural progenitor cells (NPCs), which are like these halfway pre-neuron cells. Then you pick out the best looking NPCs colonies over several cycles and reseed them, eventually getting a batch of good progenitor cells that will develop into neurons. The neurons then connect themselves into networks, and stuff happens. Jerome trained me on this, but that was like 4 years ago so I don't remember a whole lot, I just know that if you F it up and pick a bad batch, the whole colony starts turning into skin cells.

As you can tell by now, my sophistication with the biology side of things is (increasing but) rather limited. My role throughout that process was to analyze the electrical recordings from these cultures. Instead of plating the cells on a regular petri dish, they would put them on a multi-electrode array (MEA), which is just a tiny dish with some electrodes on the bottom to record action potentials from the neurons. The whole reason I was brought in was because our lab does neural signal processing, and while the software that came with the MEA system had some analysis tools, they were fairly simple metrics, like average firing rate and ISI measures.

![][expert]
*That be me.*

Knowing that we wanted to compare to EEG at some point (with no concrete idea of how at that point), we also needed to record local field potentials (LFP) for oscillations. All that meant was to record the whole MEA signal and looking at the low-frequency (LFP) and high-frequency (spikes) separately, where people typically ignore the former. Nothing fancy here, I'd just high-pass filter and get multi-unit activity and look at total population firing across all the channels, since these MEAs were not dense enough for sorting into putative single neurons. For the LFPs, I'd glance at the power spectrum for any bumps above the background and look for obvious oscillations in the autocorrelation. The pipeline sounds pretty simple, but my MATLAB code from then made me want to reach back in time and strangle myself.

![][mea]
*Cells lying on the flat electrode array emit fast action potentials and slow local field potentials, which can be recorded and analyzed separately via digital filtering. Add all the spikes together, and you get population firing.*

So that's what we did for a while: Jerome would do some recordings, and I would look at them. We didn't really find anything very interesting, and certainly no oscillations. I'd thought that since there was a small network of excitatory and inhibitory cells, that we'd at least get local gamma oscillations. Nope. All these cultures did was to spit out these sporadic and synchronized bursts, with varying delay and recruitment of neurons, but there were no fine temporal structure, and the burst of activity just decayed smoothly within a few hundred millisecond (far right panel in the figure above). It wasn't surprising, previous papers with mouse cultures have also seen this. That seemed to be the only possible spontaneous dynamic out of these networks, and it wasn't qualitatively different even if you seed the network with a lot more neurons. Jerome was wrapping up his thesis on using these 2D human iPSCs cultures to model brain diseases, and I was working on my own projects in my lab while every now and then helping him with these analyses. He'd pay me in pizza and beer, so life was just fine.

---
### 2016: From 2D Networks to 3D Brainballs
After Jerome defended his PhD, I started working with Pri and [Cleber][cleber_tw] (two post-docs in the Muotri lab) doing more or less the exact same I was doing before. They were, however, using this cool new protocol for making cerebral organoids. I had no idea what that meant. I knew what organoids were because I had a random assignment back in undergrad on designing transplantable pancreatic organoids. But as far as how the brain organoids were different from the 2D cultures, I wasn't really sure, and to be honest, I didn't really care, because I was still waiting for these networks to do something different. Spoiler alert: they did.

The main difference of the organoid protocol is that, instead of plating the iPSCs and waiting for them to form neural progenitor cells and then neurons, the iPSCs are maintained in suspension (via rotation) as stem cells. With the proper magic sauce (growth factors), the whole development process happens in this 3D form, where stem cells naturally clump together and the ones near the center start to differentiate "naturally" and eventually migrate and form these "cortical column-like" structures. I say "naturally" because, obviously, we're providing these growth factors that would be naturally produced in-embryo. This is quite a convenient breakthrough, though, because it meant you didn't have to sit at the microscope for hours scooping up nice looking NPC rosettes. Even better, this is more similar to in-vivo neurodevelopment, though in this case the cells are encouraged to specifically become cortical neurons, hence lacking any subcortical or peripheral nervous system features. Previous and subsequent studies, though, have shown the ability to grow different cell types. My understanding is that if you don't restrict them artificially, they will just sorta become whatever neurons, while you can provide growth factors to more tightly specify their fate. Or, as one expert eloquently put it in the NYT article:

![][otherstuff]

See [Lancaster 2017][lancaster2017], [Quadrato 2017][quadrato2017], and [Birey 2017][birey2017]. Especially in Quadrato 2017, where they generated organoids with a broad diversity of cells that include cortical neurons and - get this - light-sensitive retinal cells. They were able to shine light on the organoids and record evoked activity. The next time someone gives me shit for building zombie brain robots, I'm pointing them to this study first because **that** is some wild stuff. Nevertheless, while there have been several studies showing genetic and structural similarities to the developing brain, and using those as markers to study developmental disorders ([Pasca 2018][pasca2018]), the final piece - dynamics - was still missing. This is important because we know that during very early development (in-embryo and shortly after birth) the brain spontaneously generates widespread oscillatory waves. This is true in the visual pathway, and along the cortical surface ([Wong 1993][wong1993], [Garaschuk 2000][garaschuk2000]), and it's thought to aid in **experience-independent but activity-dependent** development. In other words, past the chemical signaling stages of cell migration and axon extensions, the brain **needs** activity to wire up correctly, so it generates them by itself. And this is where the real story begins.

---
### 2016-2017: Oscillating Organoids

![][orgs]
*Pretty pictures.*

So Pri grows these organoids for a few months, and she's been feeding me MEA recordings, though they always look like the strongly synchronous but monotonically decreasing population activity similar to above (which is actually from a 2-month organoid recording). One day, around September 2016, she gave me a set of recordings from ~4 months old organoids, and I run my routine population firing rate and LFP analysis. If you look over the course of the 3 minute recording (in the spike raster of the first schematic figure), it's very obvious that there are strongly synchronized bursts - the same as before.

Zooming into one of those events, though, I saw a **tiny** second bump in the population activity. A little bewildered, I checked the other 7 wells from that MEA plate, and they were all showing signs of a secondary peak. So I grab all the windows where the network bursts happen (called events in the paper), align them to the peak firing time, and overlay the population firing rate from each event on top of each other. It becomes very obvious then that the tiny oscillations are incredibly consistent: every time the event happens, it happens the same way. We wait 2 more months with bated breath, and these events become more frequent, more stereotyped, and oscillating at 3Hz for even longer. That, ladies and gentlemen, was basically the whole discovery, and Figures 2 and 3 in the paper characterizes that developmental trajectory in more detail. The same holds if you look at the LFP, where low-frequency power (background subtracted, via my prehistoric hacky FOOOF) increases over the first 6 months, but it's also really obvious by eye.

![][oscillate]

*Top row shows population spiking activity, each faint trace is one instance of an event. Bottom row shows LFP, which is qualitatively more similar to what one might measure with scalp EEG (without a skull).*

The interesting thing is that there seems to be a qualitative change again at 6 months, where these network events start to 1) become much more frequent, 2) become more variable in the latency between events, and 3) the within-event dynamic (stereotyped oscillation) breaks apart, such that each event looks different than the next (see how the faint traces at 8 months are non-overlapping). As a very rough measure of "network flexibility", we compute the coefficient of variation (CV) of the inter-event interval. This is basically Fano Factor, a measure commonly used on spike trains to measure how random a neuron is: 0 if there's no variation between spikes, so completely iso-frequency periodic, while 1 describes a Poisson process (random), and >1 describes a kind of bi-stable dynamic with bursting. The same concept applied to the network events shows that IEI CV grows monotonically, even during the first 6 months of extremely stereotyped oscillations. So the network, in some sense, acquires more noisy or flexible dynamics. I think of them like dynamical repertoires, like different template dance moves one is able to then string together to form a routine. Maybe these dynamical repertoires then piece together for different computations.

![][osc_metrics]
*Left: oscillation power measured in the LFP. Middle: randomness (or dispersal) of network events increases. Right: Proportion of different cell types over time.*

We're still not sure what exactly is enabling this class of dynamics to emerge, but Cleber (with Gene Yeo's lab) did the prerequisite gene expression analyses and show an emergence of GABAergic and glial cells at later stages. Also, if you block GABA with bicuculline at 6 months, the initiation of the network event is unaffected (even becoming a little more frequent), but the oscillations (subsequent peaks) are completely abolished. So we're very sure there is some E-I interaction going on here, but an onging question is whether GABA is inhibitory at this point, or whether the the GABAergic cells in the organoids go through the chloride reversal potential change to induce the sign flip. Okay this is getting quite technical now and I'm giving away trade secrets.

![][pac]
*Left (small panels): blocking GABA removes oscillations, but not events. Right (2 panels): phase-amplitude coupling during events.*



And because we wanted to show further relevance of this work for human neuroscience, and because these are the tools I know how to use, we looked at phase-amplitude coupling during these oscillatory events. On the one hand, cross-frequency coupling of coherent oscillations is an ongoing (albeit vague) theory of how the brain flexibly communicates across regions, so showing that the metric produces the right numbers in the organoids is interesting and doing our homework. It's also a bit of a dog-whistle for my cogneuro people, but good luck coming across this paper in Cell Stem Cell. On the other hand, high-frequency activity is a surrogate for multi-unit activity, and we know the low frequency oscillations in the LFP is **most likely** directly caused by synchronous ionic exchanges from the population spiking, so it almost has to be coupled. Indeed, we find that high-frequency activity (**not gamma oscillations**) is coupled to the phase of the 3Hz oscillation. It's still good to run that analysis, because it would've been surprising if they weren't coupled. One thing I know for sure, though, is that this is not a waveform shape artifact.

Funny tangent: after I ran the analysis on the 6-months batch of data and we were all fairly convinced that the oscillations were real, I gave a talk for the Muotri lab on our finding. But first, I had to go through what oscillations are in the context of human and system neuroscience, and the basics of spectral analysis. Despite not being trained on these techniques, these are all very smart people and they got the important bits right away. But there was one moment, I think Cleber at some point asked me how long it would take to redo these analyses on a new dataset, and I told him, well, if you want the exact same thing, it'll just take a few hours of preprocessing and remaking the figures since the code is already there. He kinda looked at me in disbelief, and said something to the effect of "you're doing magic!" **And I was like**, you people take skin cells and make stem cells and then brain cells from them, and they have the same DNA as the donor, and you think **I** do magic because I write barely legible code that can run again without failing???

![][spiderman]

Needless to say, there is a lot of mutual respect, though I always feel a little guilty that Cleber, Pri, and I are co-first authors because if you just count person-hours, god knows how long they were at the bench and microscope for, growing, tweaking, staining, and looking at these little boogers.

![][jesuscleber]
*Jesus, Cleber...*

---
### Late 2017: A Slight Digression on Publishing and Peer Review
After those initial analyses on 10 months of data, we were pretty convinced that we had something good. Again, at this point, previous organoid works have shown that, gene expression-wise and structurally, the organoids are a lot more sophisticated than the cultures we were using before, and is more similar to real development. But there were no evidence of developmentally important network activity - until now. So we felt that it really was a qualitative advance for the organoid model. Of course, where does one send a paper of qualitative advance?

I just checked my notes, it looks like we began drafting the paper in early 2017, and I think it was more or less ready by mid-2017 because I remember making figures on vacation in Cancun during spring break. We really wanted to get this out quickly because it was a simple piece of data that will augment how people do organoid work, since it's super easy to record the electrophysiology chronically and there would have been a ton of new data from different organoid protocols. Instead, 2.5 years later... We had many discussions about putting it on biorxiv at various stages of the process, and this is where the disconnect between the two labs showed up: that just wasn't the norm in stem cell biology. It wasn't our data, so we didn't push it. Plus, I understand the fear: it's a straightforward analysis and there was no real "experiment" once you have a working organoid protocol, so all someone needs is an army of undergrads to take the instructions to reproduce it in a few months time if we didn't get published right away...and we didn't.

If you look at the organoid references again from a few paragraphs above, you'll notice that there are a bunch from 2017. In fact, Nature published two articles on brain organoids in the **same** issue in May 2017. I always thought that was hilarious because we were ready to submit about 2 months later, and we had a little discussion on whether the editor would be less interested in organoids because they just put out 2 papers, or more interested because it's so hot. Hint: it wasn't the latter. Honestly, what can you say at that point? You can't sell a Ferrari to a guy that just bought two Bugattis, even though you're pretty sure the Ferrari also flies. So that was the bad luck component of it. That's fine, there are enough flashy journals to pander to, and we made the rounds in the next 12 months. It always got reviewed, and always with at least one really excited reviewer, especially about the oscillations and its implications for human neuroscience, but always rejected in the end. We had some things that could've been tighter, which really did improve after the peer review process, so I'm by no means saying peer review is useless. It's not. But there are also some **really** shitty people. Like, I pray for a Science Judgement Day where every review you've written is published with your name on it, and people can really judge for themselves how big of an asshole you are behind anonymity. Dealing with people behind curtains where they have no interest in engaging in meaningful scientific discourse is probably the single biggest reason that I'd quit science for. But it also made a huge impact on me because now I always try to not be an asshole and sign my reviews. Sign your goddamn reviews, people, these are your colleagues, and don't be an asshole. Or don't sign it, but write it as if you were going to sign it. All academia has to do is to look at some YouTube comments to realize that people will say *anything* when they're not identifiable. Just be a reasonable person, please.

Okay, rant about publication aside, there were two main things that we were sloppy about. First, the reviewers were really hung up on the fact that there were so few GABAergic cells, originally measured at just one time point (6 months I believe?), so they contended that there were effectively none, or that they were non-functional. Maybe we got carried away with interpretations, but I thought the bicuculline data was evidence enough, since it's actually a causal manipulation that showed GABA to be important. Cleber then did the single-cell RNA sequencing at multiple time points **and** measured GABA concentration changes in media via spectroscopy. Because of all this, the paper came out for the better, since now we have a better idea of how the GABAergic cells develop over time. That was what I would consider to be a reasonable complaint. Unrelated, one reviewer asked if the cells were really spiking, and claimed that MEA was not able to measure spikes. Really? Really? Aside from the fact that MEA has been used in the last 20 years for in-vitro spike measurements, we explicitly stated that we're not getting single neuron spikes, nor are we using that since I just clump all the spikes across the well anyway. I obviously cussed them out during our meeting to write the rebuttal, but Cleber had the good temperament to find 20 references where MEA was used, and got a co-author to patch a cell in the organoid. These were dark days, my friends. I was at the point where I'm happier to submit the paper somewhere than having it be accepted with revisions because that meant it was off my hands and I didn't have to do anything anymore. Sorry, really no more rants.

---
### 2018-2019: Baby EEGs and the Home Stretch
What I did have to fix was the neonatal EEG, and that was the second thing. We knew that spontaneous oscillations had some significance for human brain development, the question was what, and how do the organoid oscillations relate to it. In the first few versions of the paper, I might have stretched it a **little** far: after some PubMed and Google Scholar searches back in 2017, we realized that EEG from premature babies exhibit a very curious pattern called trace discontinu. What it looks like is bursts of high amplitude oscillations punctuating long periods of very low activity, every 10-20 seconds (middle panel below). As the baby grows to be of term, that signature disappears as the bursts become more frequent and longer, essentially connecting across the silent periods (I think?).

That's very similar to what the organoids do, so we wanted to do a formal comparison. Problem was, as good as I'd gotten at scraping free data and begging people, I had no fucking clue where to get premature baby EEG data, but we knew this was one of the necessary components to make a sell. Giving up on doing an actual comparison, we settled on just showing an example side by side with the organoid LFP. We argued internally for a while because I thought we could've just cited a study or reproduced a figure, which would've saved a lot of time. But clearly, this baby stuff went another direction. Eventually, I found this Australian EEG Database with a single dataset from a prematurely born baby, and its then curator, [Patrick Cooper][cooper_tw], helped us procure that one trace. So the last figure in the earlier versions of the paper basically amounted to "look at this trace of the organoid LFP, now look at this trace of baby EEG, they look like each other but not the adult EEG." lol.

[cooper_tw]:https://twitter.com/neurocoops

![][comparison]
*This was literally the entire final figure of the paper for some time.*



As one might imagine, the reviewers did not find this amusing, and we probably pissed them off royally with this to the point that they hated everything else. From my perspective though, it wasn't even the point of the paper, and everyone knows that it's one of those stretch claims just to reach a little higher journal-wise. Nevertheless, it was stuck in this no-man's land of either making it just a little more substantive but not knowing how, or removing it completely but losing the connection to human neuroscience. I didn't want to do it. I didn't want to do anything anymore at that point, and it was only a little bit because I was lazy. We really had no idea about where to find this **very clinical** dataset, because it had to come from preterm infants. Full-term babies 1) don't exhibit this pattern and 2) don't need clinical EEG monitoring. We certainly weren't going to collect preterm EEGs ourselves, nor were we going to set up a collaboration and wait one more year for new data to come in. It was a real pickle.

> As I'm writing this, I'm realizing that none of this process would've have been obvious from just reading the paper, so I hope you are enjoying the story.

Then came one of those moments where you're like, well, for every poo life gives you, I guess it sometimes also gives you something nice. I remember this super vividly: I think the 5 of us (me, Pri, Cleber, Alysson, and Brad) were in a meeting discussing the latest (and 5th?) rejection, where all the reviewers had taken issue with the baby EEG "comparison" (even the nice ones). So we were like, okay we have to do **something** about this now or otherwise it's going nowhere. So I reluctantly Google around for the Nth time knowing I'd find nothing, except I miraculously stumbled upon [this paper (Stevenson 2017)][stevenson2017], which was published just a few months prior to that meeting. They recorded preterm infant EEG from ~100 babies between 25-38 weeks old, and tried to predict the babies' age with their EEG features...and **they published the data!** Well, they didn't publish the raw EEG, but the table of features they computed as inputs to their prediction model. I shit you not when I say my entire PhD is built on the data (and generosity) of other people, and that's especially true here. I couldn't believe it, because it was really that easy. All I had to do then is to compute the same features in the organoid LFP and basically correlate the two. There were a few caveats. For one, I had to discard all the features that is spatial or amplitude in nature, because the organoids don't have a skull to filter the signal. Nevertheless, that left us with a handful of event timing features relating to when and how often those bursts occur.

[stevenson2017]:https://www.nature.com/articles/s41598-017-13537-3

![][mixedbag]
*There was a mixed bag of similarities.*

We compared the datasets in a couple of different ways. If you just plot age vs. feature for both organoid and infants, you'll see that the change over time for some of the features look very similar, and some not so much. For example, bursts (or SATs) per hour grow monotonically both as the babies get older and as the organoids are in culture for longer. For some features, the trend is the same, but the absolute value is different. You can measure how correlated each feature is with age, and see if those age-correlated features are the same between the babies and the organoids. This is basically one step away from a multivariate regression model, so we did just that.

![][schematic]

So I do basically what they did in that Stevenson paper, trying to use electrophysiological features to predict age. The twist, however, is that I trained a regression model on just the baby EEG features to predict their age (split into training and validation set for the regularization hyperparameters, of course). I then use that exact model to predict how long the organoids have been in culture for, or what we call "developmental time", and the punchline is that it does...not bad. In fact, it does poorly for the organoid data before 28 weeks, and becomes much better after that, and it just so turns out that we only have EEG data from babies born from 25-38 weeks. This might suggest that there is some nonlinear growth curve that cannot be linearly extrapolated to prior to 28 weeks, or it's just a happy coincidence. Either way, it's **pretty fucking incredible** that the prediction works at all for the organoid data, which, again, the regression model is completely blind to. I was geniunely surprised when this result came out. Of course, one reviewer was completely unconvinced, asking "how does one know any two datasets run under such comparison would not return similar results." I really wasn't sure what that meant, but they suggested some positive (held-out baby EEG) and negative (2D and mouse cultures) controls, for which the model of course performed as you would expect.

![][regression]
*Left panel: regression model performance on __unseen__ data. Right panel: correlation between predicted and ground-truth age/culture time for various datasets. Higher correlation means better prediction ability.*

To wrap up, I'd like to direct your attention to a magic trick I performed. As you can see, the average predicted age of the organoids (blue) follows the identity line (perfect prediction) pretty well, but the variance is horrendously large. In fact, even at 40 weeks, the model was predicting negative culture times. Oh look we discovered time traveling as well. Obviously, if we measure the model's performance with mean squared error, it would go to shit. That's why the next figure does not show MSE, but predicted-to-actual age correlation. That's the magic trick. There is, of course, a good reason for doing it this way: it comes down to the fact that we have no idea how "old" the organoids are with actual human developmental timeline as reference. In fact, when I started doing this, I had zero expectation that the model would return predictions that were anywhere near actual time, so I actually started with correlation as the metric. One organoid day could be equal to one human week, or maybe the converse is more likely, seeing how they're just brain tissue floating around with no other constraints. I did, however, expect the predicted age to at least correlate with the actual age. In other words, if you use a human ruler to measure organoids growing, the organoid should still be getting bigger, just more slowly than a human. But the surprise, once again, is that just from the features used, organoid age does seem to scale the same.

And that, my friends, are the findings from this paper (glossing over all the other important shit in the figures and supplementals that you are encouraged to look at).

---
### 2019 - ???: Enter The Matrix (No I'm Just Joking)
It's been a week since the paper came out, and if this is not the first time you're reading about this, it's more likely than not that you first read it in the [New York Times][nyt] (than from Cell Stem Cell, or, god forbid, Buzzfeed). The article does an excellent job at covering the main narrative of the science without going into technical details, but also staying very much on course in terms of **not making hyperboles** about the research, even if *space* and *robot* were mentioned. This is apparent from the title, and that was the sense I got from talking to Carl Zimmer on the phone as well.

![][NYT_title]

In fact, he explicitly asked me (and I assume Alysson as well) to comment on what the public should know about this, and he had some very pointed scientific questions that are very well worth doing follow-up experiments on. Here, I want to expand a little more on the potential implications, both from a technical and ethical point of view, obviously all surrounding the last analysis comparing organoids to baby brains.

First, one might interpret the last analysis to say that organoids at 38-weeks in culture is the same as a 38-week old baby, and one would be very wrong. The analogy here is that, if you had a pair of really foggy glasses, then an automobile would look just like a small elephant. Does that mean that the car is an elephant? Of course not. The measurement we got from the organoids are very limited, both in terms of gene expression and anatomy, not to mention that the baby EEG measurement is suffering from all the spatial mixture problems. So right off the bat, we are blurring the differences. The features we extract from the electrophysiological signal is even more limited, restricting to just the timing of the bursts. And finally, the foggiest glasses of all is the linear regression model. Yes, the model does reasonably well in predicting held-out baby EEG data, but it's squishing the already limited information we have down to an even more restricted (linear sub)space for prediction. Considering all that, we **cannot** say that organoids are like baby brains. In fact, we know that that can't be true a priori because the organoids are missing many crucial components a person has, like eyes, a thalamus, a cerebellum (really everything but a semblance of a cortex), and any of its own life-support kit (e.g., blood vessels). What we **can** say is that we found a small set of electrophysiological features - most prominently, inter-event interval of bursts - that track both baby age and organoid culture time. If we extrapolate even further, that might suggest conserved developmental trajectory of spontaneous electrical activity for naive cortical tissue. But that's about it. This should dispel all notions that we just made lab-grown baby brains. No, they're boogers that fire action potentials.

Second, and this is where it gets philosophically tricky: the organoids are **probably** not conscious. And this is not **probably** as in "I'll probably not drink tonight", but more like "the Earth is probably not going to explode for no reason tomorrow". We don't know for sure, because we can't know, but I'm pretty sure. Why? Because we don't know what constitutes the ingredients for consciousness, **and** the organoids don't oscillate at 40Hz. But people get freaked out because these are human cells, and I get that. As a knee-jerk reaction to a pop-sci story, that's probably would I would think too. And if that's all that is, then sure, I'm more than happy that this can serve as a conversation starter at the bar. Fortunately, modern neuroscience has managed to convince us that the nervous system is probably more important than your skin for defining your self and your sense of self. Unfortunately, we may be conveying that with a larger degree of certainty than we have, because it's unlikely to be the case that consciousness can arise **solely** from the nervous system. Maybe I'm drinking the West Coast embodied cognition Kool-Aid, but considering how the organoids are way less sophisticated than basically any living organism, because it cannot sense nor navigate its environment, it's hard to make a case for their consciousness. That's a long-winded way of saying "I can't say for sure but I'm pretty sure just because it's made of human cells it does not mean that they are conscious."

"But Richard it says you're putting them in robots!" Sigh...I have brought this upon myself. Okay. I can certainly attempt to write a coherent science fiction story in which we keep giving organoids ingredients to become more than just boogers, maybe becoming a fruitfly equivalent or something. I can assure you we are not there, and we won't be for **a while**. I'm punting it to the AI people here because their stuff is way more capable of actually sensing and manipulating the environment than the organoids are, as well as performing meaningful computations. Here, all I can say is that I really don't know. I don't know where the boundary lies, but I know everyone is careful of crossing it. Right now, the organoids serve as a useful model of neurodevelopment, both in terms of understanding how genetics guide early development, as well as for testing interventions to treat neural developmental disorders. If you dream really big, maybe you can imagine a point where we'll be able to answer questions about consciousness with these organoids, but I'm very doubtful of that. That being said, just as many labs across the world are moving away from macaques as an model organism due to their similarity to humans, I'm sure people will move away from organoids if it becomes clear that we're nearing an ethical boundary. The paradox here is that neuroscience is always looking for experimental organisms to model human cognition, because that's ultimately the goal. But the better the model is shown to mimic aspects of the human brain, the more incentive we have to **not experiment on it** and respect its possible sentience. I will say, though, that I'm 100% convinced that there's nothing special about a human neuron, and rodents and other animals also have consciousness, perhaps to a lesser degree than us (whatever that means). That may be an unsatisfactory "we'll see when we get to it", but I can assure you that there are [discussions][bredenoord2017] in the community already, and it's a balancing act between ethics and developing something useful.

---
To avoid ending on that slightly depressing and philosophically challenging note, I'll just say that it's been an incredible learning experience in the last 5 years working on this project. Not only did I pick up a lot of knowledge about molecular, cellular, and developmental biology, some of which are so random that I would've never thought that I'd learn (like what preterm baby EEG looks like), looking at how other people work has been insightful, especially those with different training than me. It simultaneously keeps you humble about all the seemingly magical things that you don't know, and also makes you appreciate the things you do know that other people take to be magic, even if it's just `abs(np.fft.fft())`. Even better than the paper are the friends you pick up along the way, so let's see if I can get Cleber to take us all to Fogo de Chao. As always, one study opens up way more question than it answers, and now knowing that the organoids have **the ability** to develop more variable and oscillatory dynamics, there are infinite questions to address here, a lot of which couldn't have been done with primary cultures. A lot of computational neuroscience builds theories for models of the developed cortex as if they are homogeneous blobs of excitatory and inhibitory cells, and finally, we can more or less test those theories on a fair biological implementation. I have so many questions about the computational capacity of these little brainballs, what causal roles the oscillations may play in the course of development, and how genetic abnormalities may lead to their deviations. But, so little time.

For now, though, I can finally breathe a sigh of relief that it's over. For now.

![][neanderoid]

---

[itsovercleber]:/assets/images/blog/2019-09-03-itsovercleber.png
[itsoverbrad]:/assets/images/blog/2019-09-03-itsoverbrad.png
[otherstuff]:/assets/images/blog/2019-09-03-otherstuff.png
[expert]:/assets/images/blog/2019-09-03-bwexperts.png
[mea]:/assets/images/blog/2019-09-03-mea.png
[cleber_tw]:https://twitter.com/trujillo_cleber
[rdgao]:https://twitter.com/_rdgao
[orgs]:/assets/images/blog/2019-09-03-orgs.png
[oscillate]:/assets/images/blog/2019-09-03-oscillate.png
[osc_metrics]:/assets/images/blog/2019-09-03-osc_metrics.png
[pac]:/assets/images/blog/2019-09-03-pac.png
[spiderman]:/assets/images/blog/2019-09-03-spiderman.png
[jesuscleber]:/assets/images/blog/2019-09-03-jesuscleber.png
[comparison]:/assets/images/blog/2019-09-03-comparison.png
[mixedbag]:/assets/images/blog/2019-09-03-mixedbag.png
[schematic]:/assets/images/blog/2019-09-03-schematic.png
[regression]:/assets/images/blog/2019-09-03-regression.png
[NYT_title]:/assets/images/blog/2019-09-03-NYT.png
[neanderoid]:/assets/images/blog/2019-09-03-neanderoids.png

[trujillo2019]:https://www.cell.com/cell-stem-cell/fulltext/S1934-5909(19)30337-6
[lancaster2013]:https://www.nature.com/articles/nature12517
[lancaster2017]:https://www.nature.com/articles/nbt.3906
[quadrato2017]:https://www.nature.com/articles/nature22047
[birey2017]:https://www.nature.com/articles/nature22330
[pasca2018]:https://www.nature.com/articles/nature25032
[garaschuk2000]:https://www.nature.com/articles/nn0500_452
[wong1993]:https://www.ncbi.nlm.nih.gov/pubmed/8240814
[bredenoord2017]:https://science.sciencemag.org/content/355/6322/eaaf9414

[nyt]:https://www.nytimes.com/2019/08/29/science/organoids-brain-alysson-muotri.html?smid=nytcore-ios-share&fbclid=IwAR3InZa5c68UahsgYsW3hXCq6vrEgvnN32KSN_V7-tcOUt3_2KTg7sP_Jqg


<iframe width="560" height="315" src="https://www.youtube.com/embed/7bbIQVyX6jc" frameborder="0" allow="autoplay; encrypted-media; picture-in-picture" allowfullscreen></iframe>
